delphi/example.env

```diff
@@ -21,17 +21,20 @@ SENTENCE_TRANSFORMER_MODEL=all-MiniLM-L6-v2
 
 # Database configuration
 DATABASE_HOST=localhost
```

Comment:
ideally we don't have this at all / all env config is handled at project root level